### DISCUSSION TITLE
MSSQL

### DISCUSSION THREAD

**Discussion by Zubair-KPM** [iTwin/community#30] [Q&A]
Hi,

Anyone who can help with, How to pull data in our digital twin app from MSSQL?

**Comment by pmconne** [comment]
You're going to need to be more specific.

**Reply by Zubair-KPM** [reply]
Yes sure, basically i have pulled my ACCESS CARD DATA in my MSSQL database but not able to take it on itwin app to show that data on the app in the form widget, So i need help to how  i can show that data of daily puch in my app using widget functionality
